tests/jballerina-unit-test/src/test/resources/test-src/services/service_decl_no_warnings.bal

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+public class Listener {
+    boolean initialized = false;
+    boolean started = false;
+
+    public isolated function 'start() returns error? {
+    }
+    public isolated function gracefulStop() returns error? {
```

Comment:
Let's add new lines between the methods.

Reply:
Added